examples_tests/51.WavesSimulation/ifft_x.comp

```diff
@@ -0,0 +1,62 @@
+#define _NBL_GLSL_WORKGROUP_SIZE_ 256
+
+layout(local_size_x=_NBL_GLSL_WORKGROUP_SIZE_, local_size_y=1, local_size_z=1) in;
+
+#define _NBL_GLSL_EXT_FFT_MAX_CHANNELS 3
+#define _NBL_GLSL_EXT_FFT_GET_PARAMETERS_DEFINED_
+#define _NBL_GLSL_EXT_FFT_GET_DATA_DEFINED_
+#define _NBL_GLSL_EXT_FFT_SET_DATA_DEFINED_
+#define _NBL_GLSL_EXT_FFT_GET_PADDED_DATA_DEFINED_
+#include "nbl/builtin/glsl/ext/FFT/fft.glsl"
+#include "waves_common.glsl"
+
+
+
+layout(set = 0, binding = 0, std430) buffer ANIMATED_SPECTRUM
+{
+	displacement_spectrum data[];
+} animated_spectrum;
+
+layout(set = 0, binding = 1, std430) buffer IFFT_X_OUT
+{
+	displacement_spectrum data[];
+} ifft_x_out;
+
+layout(push_constant) uniform PUSH_CONSTANTS
+{
+	nbl_glsl_ext_FFT_Parameters_t params;
+	uvec2 IMG_SIZE;
+	float TIME;
+} u_pc;
+
+#define EI(x) vec2(cos(x), sin(x))
```

Comment:
there's a function for this

Reply:
Alredy substituted, just forgot to remove the define